requirements.txt

```diff
@@ -34,6 +34,7 @@ pytz==2016.7              # via flask-restful
 requests-futures==0.9.7
 requests==2.18.1          # via requests-futures
 six==1.10.0               # via flask-appconfig, flask-cors, flask-restful, python-dateutil
+selinon==0.1.0rc5
```

Comment:
I'm not sure if this is needed as selinon is a direct dependency of worker so we don't need to state it twice. If updates will be issued, we will need to do changes twice with this.

(BTW next time you would like to introduce a dependency, place it to `requirements.in` and run `pip-compile` so deps are locked as expected).

Reply:
i added this dependency here because i need the StoragePool object to get the s3 bucket details.

Reply:
Did you experienced with some issues? This should be [already installed](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/Dockerfile#L36) in the server image as a dependency of worker.

Reply:
@jyasveer, as @fridex mentioned, selinon is a direct dependency of worker and worker is a dependency for server. So, the server container will have worker and all it's dependencies installed and you should be able to use StoragePool object directly.